src/GADM.jl

```diff
@@ -111,19 +112,18 @@ function extractdataset(path::String)
     !isnothing(dataset) ? dataset : throw(ArgumentError("failed to read dataset"))
 end
 
-
 """
-    get(country, levels...)
-Returns the MULTIPOLYGON data for the requested region.\n
-Input: ISO3 Country Code, and further subdivisions\n
+    polygon(country, levels...)
+Returns a GeoInterface compliant POLYGON/MULTIPOLYGON for the requested region.\n
+Input: ISO 3166 Alpha 3 Country Code, and further full official names of subdivisions\n
 
 ## Examples  
   
-`get("IND")` # Returns polygon of India  
-`get("IND", "Uttar Pradesh")` # Returns polygon of the state Uttar Pradesh  
-`get("IND", "Uttar Pradesh", "Lucknow")` # Returns polygon of district Lucknow  
+`polygon("IND")` # Returns polygon of India  
+`polygon("IND", "Uttar Pradesh")` # Returns polygon of the state Uttar Pradesh  
+`polygon("IND", "Uttar Pradesh", "Lucknow")` # Returns polygon of district Lucknow  
```

Comment:
Convert this to an actual Julia code section:

```raw
```julia
 # returns polygon of India
polygon("IND")

# returns polygon of the state of Uttar Pradesh
polygon("IND", "Uttar Pradesh")

# returns polygon of district Lucknow
polygon("IND", "Uttar Pradesh", "Lucknow")
```